tests/baselines/plugin_module/otio_override_adapter/plugin_manifest.json

```diff
@@ -3,9 +3,9 @@
     "adapters": [
         {
             "OTIO_SCHEMA" : "Adapter.1",
-            "name" : "cmx_3600",
+            "name" : "otio_json",
```

Comment:
@jminor, is this what you meant regarding "otio_json"?
I believe this is a part of testing the plugin system and not directly related to the cmx adapter. Therefore I just updated it to something that will remain after the cmx extarction.

Reply:
Yes, I was just suggesting that this portion of the change could be landed into main if that helps make the extract_adapters changes easier to compare. It is fine as-is though :)

Reply:
On it! Thanks!